EnglishFilter/sections/foreign.txt

```diff
@@ -4849,15 +4849,6 @@ principlesofknowledge.kr##a[href^="https://coupa.ng/"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/87516
 torrentqq96.com,torrentqq97.com,torrentqq98.com,torrentqq99.com,torrentqq100.com,torrentqq101.com,torrentqq102.com,torrentqq103.com,torrentqq104.com,torrentqq105.com,torrentqq106.com,torrentqq107.com,torrentqq108.com,torrentqq109.com,torrentqq110.com##.at-banner
 ||i.imgur.com/*.gif$domain=torrentqq96.com|torrentqq97.com|torrentqq98.com|torrentqq99.com|torrentqq100.com|torrentqq101.com|torrentqq102.com|torrentqq103.com|torrentqq104.com|torrentqq105.com|torrentqq106.com|torrentqq107.com|torrentqq108.com|torrentqq109.com|torrentqq110.com
-! https://github.com/AdguardTeam/AdguardFilters/issues/87156
-mule.co.kr##img[src^="https://cdn.mule.co.kr/data/original/"]
-mule.co.kr##div.section-leftmiddle-bottommiddle.cf
-mule.co.kr##div[class^="section-lefttop-right ad"]
-mule.co.kr#$#.section-lefttop-center { width: 100%!important; }
-mule.co.kr##div.section-lefttop-left
-mule.co.kr#$#.section-leftmiddle-topmiddle { width: 100%!important; }
-mule.co.kr##div[class$="-ad"]
-||cdn.mule.co.kr/data/original/
```

Comment:
All these rules caused this? If you remove them all, ads will appear on `mule.co.kr`.

Reply:
> If you remove them all, ads will appear on mule.co.kr.
I know. However, It is already fetched in List-KR ( https://github.com/List-KR/List-KR/commit/f443329b290155b098bcd6f0af04428e63820d4b ).

> All these rules caused this? 
No. When I contacted the reporter and connected their computer, all these rules did not cause the issue.
But, I want to remove them because I hope that there is no conflict between List-KR and AdguardFilters.

Reply:
Ok, got it. Thank you!